### PR TITLE
Clarify that call recording is not compatible with ConnectMxp action

### PR DESCRIPTION
**Pull request by christoffer-a-sinch**

### CODE REVIEW

**Review by jsiqueir-sinch — APPROVED** [review]
LGTM

File: docs/voice/voice-rest-api/voice-rest-api-callback-api.md [hunk 1]
```diff
@@ -340,6 +340,9 @@ Starts recording the call.
 
 **options** is a RecordingOptions object that specifies details about the recording. See [RecordingOptions](doc:voice-rest-api-recording#recording-options) for more details.
 
+> ❗️ `StartRecording` with `ConnectMxp` is not supported
+> Recording calls when using the action `ConnectMxp` is not supported.
```
Comment (zcausev) [inline docs/voice/voice-rest-api/voice-rest-api-callback-api.md:344]: Maybe indicate that it is not *yet* supported?

**Comment by zcausev** [comment]
One small comment, if you feel it's not needed, then LGTM.

**Comment by ulfwatt** [comment]
I was think about the same as @zcausev if we should indicate that we will in going foward support it. Not sure what I think myself on that. Otherwise it looks good to me as well

File: docs/voice/voice-rest-api/voice-rest-api-callback-api.md (hunk 1 above)
Comment (christoffer-a-sinch) [reply docs/voice/voice-rest-api/voice-rest-api-callback-api.md:344]: I think we should keep it strictly to what is currently supported.

**Pull request merged**